docs/protocol.md

```diff
@@ -465,6 +465,7 @@ code   | name                 | description
 `0x05` | unexpected error     | Request resulted in an unexpected error. The request may have been completed before the error, retry only if the request is idempotent or duplicate execution can be handled.
 `0x06` | bad request          | Request args do not match expectations, request will never be satisfied, do not retry
 `0x07` | network error        | A network error (e.g. socket error) occurred.
+`0x08` | unhealthy            | Network refused to forward request to unhealthy node.
```

Comment:
Let's use more consistent nomenclature here, "declined" rather than "refused" I think?

Also maybe we should call the name "forward_unhealthy" rather than just "unhealthy"?